lib/kitchen/verifier/pester.rb

```diff
@@ -266,6 +283,7 @@ def install_command_script
                           Write-Host "Downloaded Pester.zip"
                       }
 
+                      # Try Expand-Archive first, only fall back to COM... (PS < 5.1)
```

Comment:
Yeah, agreed. I'd just do a `Get-Command Expand-Archive` rather than a version check, just in case folks have really weird things happening on their instances. 😂